src/main/resources/static/js/signup-account.js

```diff
@@ -0,0 +1,26 @@
+document.getElementById("signup-form").addEventListener("submit", (event) => {
```

Comment:
Add 'input' event listeners to both password inputs instead (like in the settings-account.js file)